### PR TITLE
🎉(Backend) : Initialize backend Spring-boot project

### LINKED ISSUE #27: Create Spring application
Create project structure for backend spring application

### PR DESCRIPTION
**Pull request by priyo5**
Created Spring-boot project structure to be used as applications backend
Closes #27

### CODE REVIEW

**Review by ricecakesreece — APPROVED** [review]

File: Server/src/main/java/Database/TheAuPair/TheAuPairApplication.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class TheAuPairApplication implements CommandLineRunner
+{
+	private final UserRepository ur;
+	private final ParentRepository pr;
+	private final Med_AidRepository ma;
```
Comment (Ruben-Brits) [inline Server/src/main/java/Database/TheAuPair/TheAuPairApplication.java:16]: Change to CamelCase like others

File: Server/src/main/java/Database/TheAuPair/TheAuPairApplication.java [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+	private final ChildRepository cr;
+	private final ActivityRepository ar;
+	private final Au_PairRepository apr;
```
Comment (Ruben-Brits) [inline Server/src/main/java/Database/TheAuPair/TheAuPairApplication.java:19]: CamelCase

File: Server/src/main/java/Database/TheAuPair/TheAuPairApplication.java [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+	public static void main(String[] args)
+	{
+		SpringApplication.run(TheAuPairApplication.class, args);
+	}
+
+	@Override
+	public void run(String... args) throws Exception
+	{
+//		ur.save(new User("4561237814867","Reece","Stephenson","r.james.stephenson#gmail.com","1234567895","233 persheron road",true,1,"ello"));
```
Comment (Ruben-Brits) [inline Server/src/main/java/Database/TheAuPair/TheAuPairApplication.java:42]: If these comments can stay you can leave, but please remove if not needed

File: Server/src/main/java/Database/TheAuPair/TheAuPairApplication.java [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+		{
+			System.out.println(a);
+		}
+
+		for (Au_Pair ap : apr.findAll())
+		{
+			System.out.println(ap);
+		}
+
+//		uc.get("Seyuran");
```
Comment (Ruben-Brits) [inline Server/src/main/java/Database/TheAuPair/TheAuPairApplication.java:74]: Same here

**Review by Ruben-Brits — CHANGES_REQUESTED** [review]

**Review by Ruben-Brits — APPROVED** [review]

**Pull request merged**